curriculum/challenges/english/16-the-odin-project/top-introduction-to-flexbox/introduction-flexbox-question-c.md

```diff
@@ -8,12 +8,12 @@ dashedName: introduction-flexbox-question-c
 
 Somewhat confusingly, any element can be both a flex container and a flex item. Said another way, you can also put `display: flex` on a flex item and then use flexbox to arrange its children.
 
-<img src="https://cdn.statically.io/gh/TheOdinProject/curriculum/495704c6eb6bf33bc927534f231533a82b27b2ac/html_css/v2/foundations/flexbox/imgs/04.png" />
+<img src="https://cdn.freecodecamp.org/curriculum/odin-project/flex-box/flexbox-03.png" style="width: 95%; height: 95%;" alt="This image demonstrates a nested CSS Flexbox layout. It features a larger blue rectangle. Inside the red rectangle, there are two peach-colored rectangles, identified as 'flex items'. The layout shows that a flex container can nest within another flex container, and both can have their own flex items, illustrating the concept of nesting flex containers in a Flexbox layout." />
```

Comment:
Again, lose the "This image demonstrates" at the beginning. I would suggest you start with something similar to my previous suggestion and then just add the following to it:

"The middle rectangle also acts as a flex container itself, containing two smaller inner rectangles representing flex items."